lib/diagnostic.js

```diff
@@ -39,20 +58,33 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+  name: 'Octavia',
+  species: 'Gecko',
+  breed: 'Leopard',
+  vocalization: 'Moooooo'
+};
```

Comment:
Here you want to include a method that will return the `vocalization` you have added. Something like this: `vocalize: function() {return this.vocalization}`